arviz/plots/kdeplot.py

```diff
@@ -20,6 +20,7 @@ def plot_kde(
     quantiles=None,
     rotated=False,
     contour=True,
+    hpd_levels=None,
```

Comment:
All `hpd` should be changed to `hdi`, or maybe even to `levels` directly. Is there any guarantee that those areas are the minimal area with probability X?

Reply:
I've changed all `hpd_levels` to `levels` in `plot_kde`, and `hdi_levels` in `_find_hdi_contours` (which I've also renamed). I was wondering whether `hdi_probs` might be more appropriate than either `levels` or `hdi_levels`, given that it'd be similar to the argument for `arviz.hdi`?

I think the areas given are as close to minimal area as we can get with a discretised KDE?

Reply:
I agree on using hdi_probs for consistency

Reply:
I've changed it to `hdi_probs` in cf10583